specs/Spec_for_adding_user.adoc

```diff
@@ -0,0 +1,170 @@
+# User Management: Adding new User in Tendrl
+
+## Problem description
+
+This spec talks about Adding/Creating a new User to Tendrl by the Admin
+
+## Use Cases
+
+The flow goes as follows:-
+
+* A user logs in as Admin
+* Clicks on Admin tab in Navigation
+* The navigation drops down
+* Clicks on *Users*
+* Clicks on *Add* button in Users page view
+
+User will land onto https://redhat.invisionapp.com/share/KNB25OEQT#/screens/226063802[Add Users] page
+
+## Proposed change
+
+### UI changes
+
+* Add *Users* tab in Admin navigation tab
+* Add *Add User* Button in Users Page view
+* Add a form for adding a user. The form will contain following feilds:-
+** User ID
+*** Input type : text
+*** Validations:
+**** Should be Unique
+**** Minimum length: 4 characters
+**** Should be mandatory
+** First Name
+*** Input type : text
+*** Validations:
+**** Should be mandatory
+** Last Name
+*** Input type : text
+*** Validations:
+**** Should be mandatory
+** Email
+*** Input type : email
+*** Validations:
+**** Validate email ID
+**** Should be mandatory
+** Email Notification
+*** Input type : checkbox
+*** Default : Enabled
+** Password
+*** Input type : password
+*** Validations:
+**** Minimum length: 8 characters
+**** Should be mandatory
+** Confirm Password
+*** Input type : password
+*** Validations:
+**** Should be mandatory
+** Set a Role
+*** Admin
+*** Normal user
+*** Limited user
+*** Input type : Radio button
+
+*NOTE*: 
+
```

Comment:
I think we are not going to support enable/disable user. If yes we need to call out explicitly as it is there is in the design